packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -155,26 +156,25 @@ export class AuthenticatorService implements OnDestroy {
 
   /** @deprecated For internal use only */
   public get slotContext() {
-    const slotContext = {
-      ...this._facade,
-      ...this._sendEventAliases,
-    };
+    const slotContext = this._facade;
 
     return {
       ...slotContext,
       $implicit: { ...slotContext },
     };
   }
 
-  /** @deprecated For internal use only */
-  public subscribe(callback: AuthSubscriptionCallback) {
-    if (this._authService) {
-      return this._authService.subscribe(callback);
-    } else {
+  public subscribe(callback: AuthSubscriptionCallback): () => void {
+    if (!this._authService) {
       logger.error(
         'Subscription attempted before machine was created. This is likely a bug on the library, please consider filing a bug.'
       );
     }
+
+    const subscription = this._authService.subscribe(() => {
+      callback(this._facade);
+    });
```

Comment:
Here's the actual feature; we return `_facade` instead of internal state machine state value.